src/Poloniex/Spot/API/CurrencyV2.jl

```diff
@@ -0,0 +1,103 @@
+module CurrencyV2
+
+export CurrencyV2Query,
+    CurrencyV2Data,
+    currency_v2
+
+using Serde
+using Dates, NanoDates, TimeZones
+
+using CryptoAPIs.Poloniex
+using CryptoAPIs: Maybe, APIsRequest
+
+Base.@kwdef struct CurrencyV2Query <: PoloniexPublicQuery
+    # ... empty
+end
+
+struct Network <: PoloniexData
+    id::Int64
+    coin::String
+    name::Maybe{String}
+    currencyType::String
+    blockchain::Maybe{String}
+    withdrawalEnable::Bool
+    depositEnable::Bool
+    depositAddress::Maybe{String}
+    withdrawMin::Float64
+    decimals::Float64
+    withdrawFee::Float64
+    minConfirm::Int64
+end
+
+struct CurrencyV2Data <: PoloniexData
+    id::Int64
+    coin::String
+    delisted::Bool
+    tradeEnable::Bool
+    name::String
+    networkList::Vector{Network}
+    supportCollateral::Maybe{Bool}
+    supportBorrow::Maybe{Bool}
+end
```

Comment:
I think it's reasonable to rename it to `currency` without `v2`, since the method basically does the same thing, we just use the newer version here